tex/chapters/Theory.tex

```diff
@@ -0,0 +1,142 @@
+%!TEX root = ../main.tex
+
+\chapter{Theory}\label{cha:theory}
+
+\section{Analytical Fingerprints}
+There are several steps to constructing the Fingerprint of a strategy and basic knowledge of Markov Chains is required.
+An outline of the steps is as follows:
+
+\begin{enumerate}
+    \item Build the markov chain for IPD between the strategy and probe strategy.
+    \item Construct the corresponding transition matrix.
+    \item Find the steady state distribution.
+    \item Calculate the overall expected score by taking the dot product of the steady state distribution with the payoff vector given in .
+    %TODO
+    \item Plot the resulting function.
```

Comment:
Yup: keep that in mind. The numerical approach you've implemented is an approximation of that function.